Sources/TvOSPinKeyboardViewController.swift

```diff
@@ -261,9 +268,16 @@ open class TvOSPinKeyboardViewController: UIViewController {
     func deleteButtonWasPressed(sender: FocusTvButton) {
         introducedPin = String(introducedPin.dropLast())
     }
+
+    @objc
+    func menuButtonWasPressed() {
+        delegate?.pinKeyboardDidDismiss()
```

Comment:
At his point of the execution,  the pin keyboard has not dismissed yet. I think the delegate method should be called `pinKeyboardWillDismiss` or you could move this to the completion handler of `dismiss`.

Reply:
Good point. I did so.